uint16 C_RESET=0
uint16 C_LED_A=1
uint16 C_LED_F=2
uint16 C_CAM_TILT=3
uint16 C_CAM_TURN=4

uint16 cmd
int16 value
